Created workflow and about page

ogoigbe12 opened the pull request:
created a new branch called created_workflow-and-about_page in this branch newly created I add more images into the img folder and also added more svg file into the assets folder also made updates to the style.css file and also to the profile.html file also updated the README.md file.

ogoigbe12 committed:
add fest:
-update to img folder
-update to Assets folder

ogoigbe12 committed:
update to profile.html

souad988 commented on porfile.html:
```diff
@@ -56,6 +56,196 @@ <h1 class="title">
         </div>
 
     </section>
+    <section id="projects">
```
Great work @ogoigbe12  you implemented almost all this project requirements perfectly 💪🎉, Only one small thing to mention before approving your work , you need to use grid layout to align you project section container as requested in the requirements :
"In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card."

souad988 reviewed (changes requested):
Hi @ogoigbe12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![212385281-4797c58a-d358-423a-aa7c-832c22203865](https://user-images.githubusercontent.com/59707859/220198909-674283ba-ccf0-450f-b2d0-9c6477752998.gif)


## Highlights
✔No linter errors
✔Implemented almost all projects requirements as requested.

## Required Changes ♻️
_Check the comments under the review._
_ Make sure to add the acknowledgement section to your README file as you are using macroverse template you need to mention that , also it's better to replace your personel image by screenshots of your project this will make your README look professional 😊 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MaganaAsiati commented on porfile.html:
```diff
@@ -56,6 +56,196 @@ <h1 class="title">
+            </ul>
+        </div>
+    </section>
+    <section id="contact-us">
+        <h1 class="contact-form-title">
+            Contact me
+        </h1>
+        <p class="contact-form-description">
+            If you have an application you are interested in developing, a feature that you need built or a project that
+            needs
+            coding. I'd love to help with it
+        </p>
+        <form class="contact-form">
+            <input type="text" class="input" value="Samuel Ogoigbe" name="name" placeholder="Enter Name">
+            <input type="email" class="input" value="ogoigbe13@gmail.com" name="email" placeholder="Enter Email">
+            <textarea name="description" class="textarea" cols="30" rows="7"
+                placeholder="Write your message here"></textarea>
+            <button type="button" class="contact-form-button">Get in touch</button>
+        </form>
+    </section>
+    <section class="works"></section>
```
- [ ] [ OPTIONAL] Kindly remove the contact form you will work on it after with more documentation :+1:

MaganaAsiati reviewed (approved):
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @ogoigbe12  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


![clap](https://media0.giphy.com/media/j2GPljpvL8Lulm3d0X/giphy.gif)

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] [OPTIONAL] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

ogoigbe12 committed:
update to style.css

ogoigbe12 committed:
add fest:
-update to style.css
-update to README.md
-add newprofile.png image
-delete sammie.jpg

ogoigbe12 committed:
update style.css

ogoigbe12 committed:
update style.css
